init.sh

```diff
@@ -22,17 +22,19 @@ done
 if test -z "$machine" ; then
    echo "An invalid target name was given of $choice, available targets are:"
    printf '%s\n' "${targets[@]}"
-   unset machine choice
+   unset machine choice eula
+   echo "Usage: source init.sh target [accept-eula] [-f]"
```

Comment:
I would only highlight the accept eula flag and the -f flag with in the relevant if statements, however adding the unset eula here is a good addition if the user was to typo the machine choice